test/e2e/framework/pod/wait.go

```diff
@@ -421,6 +421,25 @@ func WaitForPodNameUnschedulableInNamespace(c clientset.Interface, podName, name
 	})
 }
 
+// WaitTimeoutForPodNoLongerRunningOrNotFoundInNamespace waits default amount of time (defaultPodDeletionTimeout)
+// for the specified pod to stop running or disappear. Returns an error if timeout occurs first.
+func WaitTimeoutForPodNoLongerRunningOrNotFoundInNamespace(c clientset.Interface, podName, namespace string) error {
```

Comment:
is this not the same as WaitForPodNotFoundInNamespace ?

Reply:
L545

Reply:
`WaitForPodNotFoundInNamespace` waits explicitly for `NotFound`, whereas this is either NotFound or Completed (ie. `WaitTimeoutForPodNoLongerRunningInNamespace`) so you can think of it as a combination of the two. That's how the code in storage was relying on it.

Reply:
Looking at the test that uses this, it seems like it's called after the pod is deleted? Wouldn't it be better to just wait for NotFound in that case?